packages/core/src/modules/proofs/ProofsApi.ts

```diff
@@ -368,13 +368,17 @@ export class ProofsApi<PPs extends ProofProtocol[]> implements ProofsApi<PPs> {
     }
   }
 
-  public async declineRequest(proofRecordId: string): Promise<ProofExchangeRecord> {
+  public async declineRequest(proofRecordId: string, sendProblemReport?: boolean): Promise<ProofExchangeRecord> {
```

Comment:
In general we tend to create objects for functionality like named parameters. Here for example, calling `declineProofRequest('abc-id', true)` would be quite vague. 

If we include this into 0.4.0 we can make the breaking change and the function would accept 1 parameter `DeclineRequestOptions` which is `{proofRecordId: string, sendProblemRequest?: boolean}`

Reply:
I agree with you that accepting single parameter is better but I afraid to make breaking change.

Reply:
We're just about to release 0.4.0 with a lot of breaking changes to the proofs api. Now is the time!

Reply:
I updated the method parameters to accept single `DeclineProofRequestOptions` object